dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -15,8 +15,23 @@ toc::[]
 
 To allow expansion of persistent volume claims (PVC) by {product-title} users,
 {product-title} administrators must create or update a StorageClass with
-`allowVolumeExpansion` set to `true`. Only PVCs created from that class are
-allowed to expand.
+`allowVolumeExpansion` set to `true`. Only PVCs created from that class are allowed to expand.
+
+For example, {product-title} administrators add the `allowVolumeExpansion` attribute at the bottom of the StorageClass’s configuration:
```

Comment:
IBM Style usually disagrees with using positional language, so I might say something like:

> For example, {product-title} administrators can add the `allowVolumeExpansion` attribute to the StorageClass’s configuration:

Reply:
Great, thank you for the suggestion.